Jenkinsfile

```diff
@@ -0,0 +1,45 @@
+pipeline {
+    agent { label 'linux-small' }
+    options {
+            buildDiscarder(logRotator(numToKeepStr:'25'))
+            disableConcurrentBuilds()
+            timestamps()
+            skipDefaultCheckout()
+    }
+    environment {
+        MVN_OPTS = '-Xmx1024M -Xss128M -XX:+CMSClassUnloadingEnabled -XX:+UseConcMarkSweepGC '
+        LINUX_MVN_RANDOM = '-Djava.security.egd=file:/dev/./urandom'
+    }
+    stages {
+        stage('Build') {
+            steps {
+                retry(3) {
+                    checkout scm
+                }
+                timeout(time: 20, unit: 'MINUTES') {
+                    withMaven(maven: 'M35', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${MVN_OPTS} ${LINUX_MVN_RANDOM}') {
```

Comment:
I'll have to check if we actually have an `M35` maven version defined, I know there is a `Maven 3.5.4` label defined though

Reply:
We do and we use it elsewhere as well 👍